### ISSUE TITLE
owner of the agent config file

### ISSUE THREAD

**Issue by ksivask** [nginxinc/nginx-amplify-agent#115]
https://github.com/nginxinc/nginx-amplify-agent/blob/5dcb87deef6c2773f95278c18e5ce35da5a12958/packages/install.sh#L549

per this line, the owner is set to 'nginx', but shouldn't the owner be actually set to ${amplify_user} ?

**Comment by gshulegaard** [comment]
Good catch, confirmed it is still an issue with our internal master.  Will open a ticket.

**Comment by IAM-db** [comment]
This is not the owner I had a Moto e5 Plus, Samsung J3 Emerge, LG Tribute, windows phone, Samsung Galaxy 6, moto g(7) play, kyrocera water proof all were Samsung null Target. My data was leaked in 2018 - 2019 by William David Smith, Jessica Rowland Pawless Seal, Dimitry Tzanous.
